### PR TITLE
added code coverage for PlatformEventDistributorDIModule

### PR DESCRIPTION
**Pull request by madiraza**
This change is [Reviewable](https://reviewable.io/reviews/apex-enterprise-patterns/at4dx/53)

### CODE REVIEW

File: sfdx-source/core/test/classes/framework-platform-event-distributor/PlatformEventDistributorDIModuleTest.cls [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            EventBus__c = AT4DXMessage__e.SobjectType.getDescribe().getName(),
+            Event__c = 'TestOfPlatformEventDistributor',
+            Execute_Synchronous__c = true
+        );
+
+        PlatformEventDistributorDIModule dimod = new PlatformEventDistributorDIModule();
+        PlatformEventDistributorDIModule.registeredSubscribers = new List<PlatformEvents_Subscription__mdt> { pes, pes2 };
+        dimod.configure();
+
+        System.assertEquals(
```
Comment (stohn777) [inline sfdx-source/core/test/classes/framework-platform-event-distributor/PlatformEventDistributorDIModuleTest.cls:31]: @madiraza @bspeelm 
Please touch the indenting in this Assert, and I'm good.

**Review by stohn777 — CHANGES_REQUESTED** [review]
Thank you greatly for your help with code coverage and testing.

File: sfdx-source/core/test/classes/framework-platform-event-distributor/PlatformEventDistributorDIModuleTest.cls (hunk 1 above)
Comment (bspeelm) [reply sfdx-source/core/test/classes/framework-platform-event-distributor/PlatformEventDistributorDIModuleTest.cls:31]: @stohn777 no problem should look better now.

File: sfdx-source/core/test/classes/framework-platform-event-distributor/PlatformEventDistributorDIModuleTest.cls [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
```
Comment (stohn777) [inline sfdx-source/core/test/classes/framework-platform-event-distributor/PlatformEventDistributorDIModuleTest.cls:31]: @bspeelm  After a more attentive look, it seems this test is more about code coverage, which is okay, but may I suggest a slightly different Assert that actually means something?  First, externally expose the 'warningMessage' variable in the DIModule.  Second Assert that it's blank after valid CMD records are provided to the class.  With that, more tests could provide invalid CMD records and Assert the 'warningMessage' is appropriate.

Comment (bspeelm) [reply sfdx-source/core/test/classes/framework-platform-event-distributor/PlatformEventDistributorDIModuleTest.cls:108]: @stohn777 Let me know what you think of this. I exposed the warning message through a private static variable. I suppose the warningMessage itself could be completely moved out of the configure() method but I didn't want to be destructive.

Comment (stohn777) [reply sfdx-source/core/test/classes/framework-platform-event-distributor/PlatformEventDistributorDIModuleTest.cls:108]: @bspeelm That will certainly do the job.

Personally I wouldn't use the "magic string" as the expected value; only Assert that it's populated or not.  BUT we can consider cleaning-up the messaging functionality later.

**Review by stohn777 — APPROVED** [review]

**Review by ImJohnMDaniel — APPROVED** [review]

**Pull request merged**